scripts/npm/update-owners.sh

```diff
@@ -16,10 +16,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-set -e
```

Comment:
Why removing the `set -e` the fail early it's always a good thing, what's the reason for removing it?